src/blog/templates/blog/blog_page.html

```diff
@@ -37,13 +37,13 @@ <h1 class="bold text-4xl mb-0 text-black">
                 {{ page.body|richtext }}
             </article>
 
-            <div class="flex flex-row flex-wrap p-1">
+            <div class="p-4 grid md:grid-cols-2 lg:grid-cols-3 gap-4">
                 {% for item in page.gallery_images.all %}
-                    <div class="flex-col p-1" style="width: 320px">
-                        {% image item.image fill-320x240 %}
-                        <p>{{ item.caption }}</p>
-                    </div>
-                {% endfor %}    
+                <div class="bg-white rounded-xl overflow-hidden drop-shadow group hover:rotate-3 transition-transform">
+                    {% image item.image fill-703x408 %}
```

Comment:
Why these specific dimensions? 😸

Reply:
Because it looks like this with 480x360
![Uploading Scherm­afbeelding 2024-06-02 om 10.00.03.png…]()